api/v6/report_server.thrift

```diff
@@ -25,6 +25,22 @@ struct RunData {
 }
 typedef list<RunData> RunDataList
 
+struct RunHistoryData {
+  1: i64       runId,    // Unique id of the run.
```

Comment:
Do we need to return the `runId` the client calls the api with a runId `getRunHistory(1: i64 runId)`?

Reply:
Yes, we need it, because we can call the `getRunHistory` function without run id. In this case we get back history data for all runs. (See Run history tab under All reports).